build.py

```diff
@@ -159,6 +159,11 @@ def dequote(value):
                     '--target=.',
                     '--requirement=requirements.txt',
                 )
+    if runtime.startswith('node'):
+        packageJson = os.path.join(temp_dir, 'package.json')
+        if os.path.exists(packageJson):
+            with cd(temp_dir):
+                run('npm','install',)
```

Comment:
Thoughts on using `--prod` so the lambda package does not include dev dependencies?

```suggestion
                run('npm','install','--prod',)
```

Reply:
Sounds good